x-pack/legacy/plugins/canvas/i18n/components.ts

```diff
@@ -374,4 +380,203 @@ export const ComponentStrings = {
         defaultMessage: 'Reset',
       }),
   },
+  WorkpadLoader: {
+    getClonedWorkpadNameSuffixLabel: () =>
+      i18n.translate('xpack.canvas.workpadLoader.clonedWorkpad', {
+        defaultMessage: 'Copy',
+        description:
+          'This suffix is added to the end of the name of a cloned workpad to indicate that this ' +
```

Comment:
Is concatenation like this how we should handle ReallyReallyReallyLongStrings™? Or should we just let them flow?

Reply:
Should we maybe do "Copy of <workpad-name>"?